package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clr-addons",
-  "version": "7.4.0",
+  "version": "7.4.1",
```

Comment:
pls raise version to 7.5.0, as I updated clarity dependency to 3.1.1
to avoid this in future, just make a separate version commit